dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -0,0 +1,438 @@
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			showAge={showAge}
+			headlineSize="huge"
+			headlineSizeOnMobile="large"
+			imageUrl={trail.image}
+			imagePosition={'top'}
+			imagePositionOnMobile={'top'}
+			supportingContent={trail.supportingContent?.slice(0, 4)}
+			supportingContentAlignment={'horizontal'}
+		/>
+	);
+};
+
+/**
+ * ._________________________.
+ * |          ###############|
+ * |          ###############|
+ * |__________###############|
+ * Card designed to take up 75% of the container, with media that takes up 50%
+ *
+ * Options:
+ *  - Large headline (large on mobile)
+ *  - Large image on the right (top on mobile)
+ *  - Trail text
+ *  - Up to 3 supporting content items, 1-2 aligned vertical, 3 aligned horizontal
+ */
+export const Card75Media50Right = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			showAge={showAge}
+			trailText={trail.trailText}
+			supportingContent={trail.supportingContent?.slice(0, 3)}
+			supportingContentAlignment={
+				trail.supportingContent && trail.supportingContent.length > 2
+					? 'horizontal'
+					: 'vertical'
+			}
+			imagePosition="right"
+			imageSize="large"
+			imagePositionOnMobile="top"
+			headlineSize="large"
+		/>
+	);
+};
+
+/**
+ * ._________________________.
+ * |###############          |
+ * |###############          |
+ * |###############__________|
+ * Card designed to take up 75% of the container, with media that takes up 50%
+ *
+ * Options:
+ *  - Large headline (large on mobile)
+ *  - Large image on the left (top on mobile)
+ *  - Trail text
+ *  - Up to 3 supporting content items, 1-2 aligned vertical, 3 aligned horizontal
+ */
+export const Card75Media50Left = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			showAge={showAge}
+			trailText={trail.trailText}
+			supportingContent={trail.supportingContent?.slice(0, 3)}
+			supportingContentAlignment={
+				trail.supportingContent && trail.supportingContent.length > 2
+					? 'horizontal'
+					: 'vertical'
+			}
+			imagePosition="right"
+			imageSize="large"
+			imagePositionOnMobile="top"
+			headlineSize="large"
+		/>
+	);
+};
+
+/**
+ * ._________.
+ * |#########|
+ * |#########|
+ * |_________|
+ * Card designed to take up 25% of the container, with media that takes up 25%
+ *
+ * Options:
+ *  - Medium headline (medium on mobile)
+ *  - Small image on the top (left on mobile)
+ *  - No trail text
+ *  - Up to 2 supporting content items, always aligned vertical
+ */
+export const Card25Media25 = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			supportingContent={trail.supportingContent?.slice(0, 2)}
+			containerPalette={containerPalette}
+			showAge={showAge}
```

Comment:
I think I'm missing something here. Why are we not  passing `headlineSize`, `headlineSizeOnMobile`, `imageSize`, `imagePosition`, `imagepositionOnMobile` and `supportingContentAlignment`?

Reply:
I have the same question for other wrappers, e.g. `Card25Media25Tall. etc.